### DISCUSSION TITLE
Popup animations for opening/dismissing?

### DISCUSSION THREAD

**Discussion by LeoJHarris** [CommunityToolkit/Maui#2079] [New Feature Discussions]
Hi there, first of all thanks heaps team for putting together this toolkit and the continued effort on making it amazing!!

As a newly arrived MAUI developer from Xamarin we were previously using Prism's dialog service which as I understand took advantage of the Xamarin Popups library, we are in the process of migrating from dialogs to MAUI community popups but liked the animations prism dialogs provided which we feel gave the app a smoother and more polished finish when loading/dismissing dialogs. putting my 2 cents worth forward that I think this would be a great addition if not animations currently supported for popups such as fading, slide up/down for dialogs appearing and dismissing.

**Comment by IeuanWalker** [comment]
@LeoJHarris pretty sure prism dialogs is a custom implementation by themselves, and is based on ContentPage.

A similar open source alternative would Mopups - https://github.com/LuckyDucko/Mopups

Was using the previous XF version in production for years, and now use this in our Maui migration, and all works perfectly.

**Reply by LeoJHarris** [reply]
@IeuanWalker We started exploring `Mopups` but our preference is and becoming more so since migrating to MAUI is to trim down on the amount of additional 3rd party SDK's we integrate into our apps. Migrating to MAUI has given us a bit of a restart as what we do and do not integrate going forward, trying to leverage off this community toolkit as much as possible would be ideal.

**Reply by bijington** [reply]
The toolkit doesn't provide the ability to customise the animations when being presented or hidden. Due to the fact that these are using the platform implementations I can't imagine it would be an easy thing to support. 

I would recommend using Mopups as per @IeuanWalker suggestion if you need to customise the animation

**Reply by LeoJHarris** [reply]
@bijington thanks for letting me know, @IeuanWalker thanks for the suggestion, some food for thought for now.
